powerfulseal/cli/__main__.py

```diff
@@ -74,6 +74,12 @@ def add_ssh_options(parser):
             '(for example for minikube) you can override it here'
         )
     )
+    args_ssh.add_argument(
+        '--use-private-ip',
+        default=False,
```

Comment:
I was going to suggest defaulting this to true for backwards compatibility, but since you modified the existing drivers to use the same value for the external IP, I guess that's satisfied anyway.